app/src/androidTest/java/org/schabi/newpipe/local/playlist/LocalPlaylistManagerTest.kt

```diff
@@ -0,0 +1,85 @@
+package org.schabi.newpipe.local.playlist
+
+import androidx.room.Room
+import androidx.test.core.app.ApplicationProvider
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.Timeout
+import org.schabi.newpipe.database.AppDatabase
+import org.schabi.newpipe.database.stream.model.StreamEntity
+import org.schabi.newpipe.extractor.stream.StreamType
+import org.schabi.newpipe.testUtil.TrampolineSchedulerRule
+import java.util.concurrent.TimeUnit
+
+class LocalPlaylistManagerTest {
+
+    private lateinit var manager: LocalPlaylistManager
+    private lateinit var database: AppDatabase
+
+    @get:Rule
+    val trampolineScheduler = TrampolineSchedulerRule()
+
+    @get:Rule
+    val timeout = Timeout(10, TimeUnit.SECONDS)
+
+    @Before
+    fun setup() {
+        database = Room.inMemoryDatabaseBuilder(
+                ApplicationProvider.getApplicationContext(),
+                AppDatabase::class.java
+        )
+                .allowMainThreadQueries()
+                .build()
+
+        manager = LocalPlaylistManager(database)
+    }
+
+    @After
+    fun cleanUp() {
+        database.close()
+    }
+
+    @Test
+    fun createPlaylist() {
+        val stream = StreamEntity(
+                serviceId = 1, url = "https://newpipe.net/", title = "title",
+                streamType = StreamType.VIDEO_STREAM, duration = 1, uploader = "uploader"
+        )
+
+        val result = manager.createPlaylist("name", listOf(stream))
+
+        // This should not behave like this.
+        // Currently list of all stream ids is returned instead of playlist id
```

Comment:
Why? Sholdn't the behaviour just depend on what is needed? I see [here](https://github.com/TeamNewPipe/NewPipe/blob/e75d0de135425de40f47bd9299f496b615a14253/app/src/main/java/org/schabi/newpipe/local/dialog/PlaylistCreationDialog.java#L62-L65) that the result is not used at all, so `createPlaylist()` could as well just be a `Completable` that throws an exception on an error, instead of a `Maybe`. But this is obviously for another PR.

Reply:
For me if I have a create operation I want to get the ID of the just created entity. When i first saw the method signature I thought I could create multiple playlists, because why otherwise would it return mutliple IDs. Here it returns the IDs of the streams which I passed in.  

In tests for create operations I usually also fetch the just created entity from the database, which I can't do here because I don't have the ID.

Reply:
Oh, I see. So you would return the ID of the just created entity for consistency and testing purposes. Ok